api/db/seeds/data/common/tooling/session-tooling.js

```diff
@@ -793,6 +793,10 @@ function _buildSession({
   });
 }
 
+function _generateEmail({ sessionId, index }) {
+  return `firstname-lastname-${index}-${sessionId}@example.net`;
```

Comment:
Encore plus simple ?

```suggestion
  return `user-${index}-${sessionId}@example.net`;